packages/i18n-calypso/cli/test/examples/i18n-test-examples.jsx

```diff
@@ -65,21 +65,23 @@ corners.` );
 	content = i18n.translate( 'single test2', 'plural test2', { count: varCount } );
 
 	// test the ability to break strings to multiple lines
+	// prettier-ignore
 	content = <div>{ i18n.translate(
 		"This is a multi-line translation with \"mixed quotes\" " +
 		'and mixed \'single quotes\''
 	) }</div>;
 
-	content = this.translate( 'The string key text',
-		{ 'context': 'context with a literal string key' } );
+	content = this.translate( 'The string key text', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

Reply:
@Automattic/i18n Is there a way to ignore strings from translation? Should we be concerned about this?

Reply:
Sounds like we can just leave them, thanks 👍

Reply:
I'd rather them removed but you probably don't need the overhead here so: #30143